README.md

```diff
@@ -1,41 +1,159 @@
-## Open Container Initiative Distribution Specification
+# OCI Distribution Specification
 
-The [Open Container Initiative](https://www.opencontainers.org/) develops specifications for standards on Operating System process and application containers.
+The OCI Distribution Spec project defines an API protocol to facilitate and standardize the distribution of content.
 
-The specification can be found [here](https://github.com/opencontainers/distribution-spec/blob/master/spec.md).
+**[The specification can be found here](spec.md).**
 
-### Table of Contents
+This repository also provides [Go types](specs-go), and [registry conformance tooling](conformance).
+The Go types and validation should be compatible with the current Go release; earlier Go releases are not supported.
 
-- [Code of Conduct][code-of-conduct]
+Additional documentation about how this group operates:
+
+- [Contributing](CONTRIBUTING.md)
+- [Governance](GOVERNANCE.md)
+- [Maintainers' Guide](MAINTAINERS_GUIDE.md)
 - [Releases](RELEASES.md)
-- [Charter][charter]
-- [Frequently Asked Questions](faq.md)
 
-### Use Cases
+The _optional_ and _base_ layers of all OCI projects are tracked in the [OCI Scope Table](https://www.opencontainers.org/about/oci-scope-table).
+
+## Distributing OCI Images and other content
+
+The OCI Distribution Spec is closely related to the [OCI Image Format Spec project](https://github.com/opencontainers/image-spec),
+the [OCI Runtime Spec project](https://github.com/opencontainers/runtime-spec),
+and the [OCI Artifacts project](https://github.com/opencontainers/artifacts).
+
+The Image Format Specification strictly defines the requirements for an OCI Image (container image), which consists of
+a manifest, an optional image index, a set of filesystem layers, and a configuration.
+The schema for OCI Image components is fully supported by the APIs defined in the Distribution Spec.
+
+The OCI Runtime Specification defines how to properly run a container "[filesystem bundle](https://github.com/opencontainers/runtime-spec/blob/master/bundle.md)"
+which fully adheres to the OCI Image Format. The Runtime Spec is relevant to the Distribution Spec in that they both support OCI Images,
+and that container runtimes use the APIs defined in the Distribution Spec to fetch pre-built container images and run them.
+
+The Distribution Spec is also designed generically enough to be leveraged as a distribution mechanism for
+any type of content. The format of uploaded manifests, for example, need not necessarily adhere to the OCI Image Format
+so long as it references the blobs which comprise a given artifact.
+
+The OCI Artifacts project is an effort to provide guidance on how to
+properly define and distribute content using the Distribution Spec for artifacts which are not container filesystem bundles,
+in a way that is mostly compatible with the existing schemas defined in the Image Format Spec.
```

Comment:
This is all new content. Tried to give the bigger picture

Reply:
For each paragraph in this section, I'd make the the specification being reference links, too.

```md
The [OCI Image Format Specification] ...

The [OCI Runtime Specification] ...

The [OCI Distribution Spec] ...

The [OCI Artifacts] ...

[OCI Image Format Specification]: https://github.com/opencontainers/image-spec
[OCI Image Runtime Specification]: https://github.com/opencontainers/runtime-spec
[OCI Image Distribution Specification]: https://github.com/opencontainers/distribution-spec
[OCI Image Artifacts Specification]: https://github.com/opencontainers/artifacts

```

Reply:
Also notice that I tried to be consistent with usage of "OCI" when formally naming things.